.github/workflows/swift.yml

```diff
@@ -4,13 +4,18 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: macos-latest
+    runs-on: macos-11.0
```

Comment:
Is macos-11 not macos-latest? if it isn't please open an issue to go back to latest when they upgrade it.

Reply:
macOS-11 is indeed not the default yet. It's still listed as experimental and will probably become the default at some point in the future.

Reply:
I'm not sure if and when GitHub is planning to make `macos-11.0` synonymous with `macos-latest`.

See the following note on the specification page for GitHub-hosted runners:
> Note: The MacOS 11.0 virtual environment is currently provided as a preview only. The macos-latest YAML workflow label still uses the MacOS 10.15 virtual environment.

Reply:
Ok. Please open an issue to go back to latest when they upgrade it.

Reply:
Done, see #13